Makefile

```diff
@@ -168,7 +168,7 @@ bots:
 # when you start a new project, use the latest release, and update it from time to time
 test/common:
 	flock Makefile sh -ec '\
-	    git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 748cfe36086cdd7069c698d0193a1b4751a1b80e; \
+	    git fetch --depth=1 https://github.com/cockpit-project/cockpit.git b49422a9c82858df0a0df74e7b8207472a2e9110; \
```

Comment:
We just released 262 - maybe we can use that already?

Reply:
Good point!

Reply:
Done.